### DISCUSSION TITLE
save chapter state

### DISCUSSION THREAD

**Discussion by devvythelopper** [Goodwy/PlayBooks#88] [Ideas]
There is quite a number of audiobooks which are collections of some sort (essays, short stories, even novels). It would be great to have an option to treat these books more like podcasts, where the chapters are episodes and where you can resume any episode whenever you like (i.e. the chapter not starting from the beginning when you select it, but continueing where you left off).

I like playbooks easy to use and non-cluttered UI

**Comment by Goodwy** [comment]
If each file is a separate chapter, you can add and

**Reply by devvythelopper** [reply]
thank you for your quick reply!

yeah, I could work around it, but I don't want to. Things get too cluttered if you turn a book with 81 short stories into 81 audiobooks. In the book too, they are categorized by story type. It would be the same in the real world... you don't want 81 booklets flying around when you can have a single book, where they are organized inside. Also, I mean it is work: The whole point of computer science is to let computers do the work for you. I want a simple checkbox where I can say that I want to treat the chapters of a specific book as separate entities. 

I appreciate your work btw. and if PlayBook becomes my audiobook app, I'll be sure to support you. I can also give in advance if that is necessary to fund development.

Btw. strike the podcast idea. I think it is good that playbook only focuses on listening and bookmarks. I love that it is so uncluttered.
